### PR TITLE
all: gofmt -w -r 'interface{} -> any' .

### PR DESCRIPTION
**Pull request by sashamelentyev**
Use any instead of interface{} because file.d use go 1.18

### CODE REVIEW

File: plugin/output/s3/mock/s3.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ func (m *MockObjectStoreClient) BucketExists(bucketName string) (bool, error) {
 }
 
 // BucketExists indicates an expected call of BucketExists.
-func (mr *MockObjectStoreClientMockRecorder) BucketExists(bucketName interface{}) *gomock.Call {
```
Comment (vadimalekseev) [inline plugin/output/s3/mock/s3.go:47]: Is this the output of MockGen?

// Code generated by MockGen. DO NOT EDIT.

Comment (sashamelentyev) [reply plugin/output/s3/mock/s3.go]: My bad. I regenerate mock

File: plugin/output/postgres/mock/postgres.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ func (mr *MockPgxIfaceMockRecorder) Close() *gomock.Call {
 }
 
 // Query mocks base method.
-func (m *MockPgxIface) Query(ctx context.Context, sql string, args ...interface{}) (pgx.Rows, error) {
+func (m *MockPgxIface) Query(ctx context.Context, sql string, args ...any) (pgx.Rows, error) {
```
Comment (vadimalekseev) [inline plugin/output/postgres/mock/postgres.go:51]: Same

Comment (sashamelentyev) [reply plugin/output/postgres/mock/postgres.go]: Hmm. Revert this

**Review by vadimalekseev — APPROVED** [review]
LGTM!

Thanks for the first contribution! :tada: 

Next time please read our contribution documentation: https://github.com/ozontech/file.d/blob/master/CONTRIBUTING.md#pull-requests

**Comment by ansakharov** [comment]
@sashamelentyev thanks for contribution.

**Pull request merged**